src/lib/skip.ts

```diff
@@ -238,3 +239,14 @@ export const fetchTransferStatus = ({
 }) => {
   return fetchSkipStatus({ transactionHash: transactionId, chainId: fromChainId, baseUrl });
 };
+
+export const skipClient = new SkipClient({
+  endpointOptions: {
+    getRpcEndpointForChain: async (chainId: string) => {
+      if (chainId === 'noble-1') return 'https://noble-yx-rpc.polkachu.com/';
+      if (chainId === 'dydx-mainnet-1') return 'https://dydx-ops-rpc.kingnodes.com';
+      if (chainId === '43114') return 'https://api.avax.network/ext/bc/C/rpc';
```

Comment:
Maybe we can get these from config as well. 

Also interesting that we need to set avax specifically 👀

Reply:
oh, sorry this was just testing code. this should be replaced by the `useSkipClient` hook. forgot to hook that up

Reply:
actually, i'm unsure how this code even snuck in here in the first place 👀 must have accidentally moved it from another branch. oops!